src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -292,9 +322,12 @@ class Simplex_tree {
 
   /** \brief Returns a range over the vertices of the simplicial complex. 
    * The order is increasing according to < on Vertex_handles.*/
+  Const_complex_vertex_range complex_vertex_range() const {
+    return Const_complex_vertex_range(boost::make_transform_iterator(root_.members_.begin(), return_first()),
+                                      boost::make_transform_iterator(root_.members_.end(), return_first()));
+  }
   Complex_vertex_range complex_vertex_range() {
```

Comment:
Why does this one have a non-const version (while most others don't)?

Reply:
There isn't really a reason, I removed the non-const version.